Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2.7
+FROM python:2.7-alpine
```

Comment:
Something tells me, this will break installs with only python:2.7

Reply:
No this just means use Alpine instead of Debian. It's reverting the change to Debian that was made to fix the Notifications problem, that we fixed with the stack_size config.